lib/mint/http1.ex

```diff
@@ -265,6 +265,8 @@ defmodule Mint.HTTP1 do
   @doc """
   See `Mint.HTTP.stream_request_body/3`.
 
+  In HTTP/1, sending an empty chuunk is a no-op.
```

Comment:
```suggestion
  In HTTP/1, sending an empty chuunk is a no-op, use `:eof` to stop the request body streaming.
```

Maybe something like this? Should we add this to `HTTP` docs as well?